src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/logging/CheckLoggingSpec.groovy

```diff
@@ -86,4 +88,26 @@ class CheckLoggingSpec extends HealthCheckSpecification {
                     "Storm should generate a warning message about not being able to find the flow"
         }
     }
+
+    def "Check push discovery package logging"() {
+        given: "Two port(with/without ISL)"
+        def sw = topology.activeTraffGens*.switchConnected.first()
+        def portWithIsl = topology.getRelatedIsls(sw).first().srcPort
+        def portWithoutIsl = 10 //default port for traffgen
+
+        when: "Retrieve logs for port with ISL for last 1 minute"
+        def result
+
+        then: "There should be 1 message of push discovery for port without ISL and more than 1 for port with ISL"
+        Wrappers.wait(discoveryFailedInterval + WAIT_OFFSET) {
+            result = elastic.getLogs(new ElasticQueryBuilder().setTags(KildaTags.FLOODLIGHT)
+                    .setLevel("INFO").setTimeRange(60).build())
+            assert result.hits.hits.findAll { hit ->
+                hit.source.message.toLowerCase().contains(pushDiscoveryMsg("$sw.dpId-$portWithoutIsl"))
+            }.size() == 1
+            assert result.hits.hits.findAll { hit ->
+                hit.source.message.toLowerCase().contains(pushDiscoveryMsg("$sw.dpId-$portWithIsl"))
+            }.size() > 1
```

Comment:
can this number be calculated based of discovery interval and requested timerange?

Reply:
I tried to do it, but I had no luck

I mean I see needed amount of logs in kibana, but can't get them by `elastic.getLogs(new ElasticQueryBuilder().setTags(KildaTags.FLOODLIGHT).setLevel("INFO").setTimeRange(60).build())`